meta/types.go

```diff
@@ -30,7 +30,8 @@ var Types = []TypeInfo{
 	TypeInfo{Prefix: "NullString", Literal: "sql.NullString"},
 	TypeInfo{Prefix: "NullTime", Literal: "NullableTime"}, // TODO(shutej): test
 	TypeInfo{Prefix: "String", Literal: "string"},
-	TypeInfo{Prefix: "Time", Literal: "time.Time"}, // TODO(shutej): test
+	TypeInfo{Prefix: "StringArray", Literal: "[]string"}, // TODO(benbardin): test
```

Comment:
i don't know that lib/pq will know how to handle this, either, though.  this is a pretty esoteric use of postgres features